content/next-docs/release-notes/release-notes-1.9.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Release 1.9
+description: 'cert-manager release notes: cert-manager v1.9'
+---
+
+## v1.9.0
+
+cert-manager 1.9 adds alpha support for using cert-manager `Certificate`s in scenarios where the ordering of the RDN sequence that constitutes an X.509 certificate's subject needs to be preserved, improves ability to configure the `Certificate` created via ingress-shim using annotations on the `Ingress` resource and introduces various changes/improvements in contributor flow, most importantly defines a new make based contributor workflow.
+
+### Major Themes
+
+#### `LiteralCertificateSubject` feature
+
+cert-manager's `Certificate` allows to configure the subject fields of the X.509 certificate via `spec.subject` and `spec.commonName` fields. The [X.509 spec](https://datatracker.ietf.org/doc/html/rfc5280#section-4.1.2.6) states that the subject is an (ordered) sequence of Relative Distinguished Names (RDN). cert-manager does not strictly abide by the spec when encoding the subject fields from `Certificate`'s spec, for example, the order of the RDN sequence may not be preserved.
```

Comment:
RDN is used on line 8 and expanded on line 14... My natural inclination is to put the expansion on 8, or at least a forward link to a thing...